packages/pages/tests/fixtures/src/functions/http/[param].js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
Why is this needed?

Reply:
It's there because `loadFunctionModules()` in `loader.ts` has the `transpile` argument so this file is to test `transpile = false`. `tests/fixtures/src/template.jsx` does the same for the template loader.

But, we actually don't load transpile functions anymore -- since the dist directory structure is different it would fail to load. The stuff with copying to a `.js` file yesterday is what replaces this transpiled load.

So, this file needs to exist if we want to test `transpile = false` but I suppose we could also remove that as an argument if it is never used?